test/ModAttachr/modAttachr.ts

```diff
@@ -0,0 +1,36 @@
+import { ModAttachr } from "../../src/ModAttachr";
+
+import { mochaLoader } from "../main";
+
+import { EventNames } from "../../src/EventNames";
+
+import {
+    IMod
+} from "../../src/IModAttachr";
+
+mochaLoader.it("Creates a mod and fires the contained events", (): void => {
+    // Arrange
+    const eventNames: EventNames = new EventNames();
+    const dummyMod: IMod[] = [{
```

Comment:
It would be a little cleaner to make `dummyMod` an `IMod` and have the constructor below take it as `[dummyMod]`.